profiles/Jonas-setup.yaml

```diff
@@ -177,7 +177,6 @@ gear_sets:
 prep_scaling_factor: 0.95
 cambrinth: cambrinth armband
 cambrinth_cap: 32
-held_cambrinth: false
```

Comment:
Are you going to also remove the `held_cambrinth` validator?

Reply:
That would make sense! Will do

Reply:
Done